.github/ISSUE_TEMPLATE/0_bug.yml

```diff
@@ -13,7 +15,7 @@ body:
 
         Since there are already a lot of open issues, please also take a moment to search existing ones to see if your bug has already been reported. If you find something related, please upvote that issue and provide additional details as necessary.
 
-        💐 Thank you for helping to make Conda better. We would be unable to improve Conda without our community!
+        💐 Thank you for helping to make `conda/infrastructure` better. We would be unable to improve `conda/infrastructure` without our community!
```

Comment:
@kenodegard I _think_ that should be conda/conda?

Reply:
grrr this is a side effect of these other repos still using the old workflow

Reply:
https://github.com/conda/infrastructure/pull/978 should fix this issue